L1Trigger/L1THGCal/python/customTowers.py

```diff
@@ -25,7 +25,7 @@ def custom_towers_etaphi(process,
     return process
 
 
-def custom_towers_map(process, 
+def custom_towers_map(process,
         towermapping='L1Trigger/L1THGCal/data/tower_mapping_hgcroc_eta-phi_v0.txt',
```

Comment:
Maybe the default parameters here would need to be updated (`towermapping`, `minEta`, `maxEta`)